MAINTAINERS.md

```diff
@@ -0,0 +1,143 @@
+| Can choose Milestones for specific component                                                                                                |            ✗            |            ✓           |            ✓            |
+| Make a decision on project's Milestones during planning                                                                                     |            ✗            |            ✗           |            ✓            |
+| Can propose new RFC or<br> participate in review of existing RFC                                                                            |            ✓            |            ✓           |            ✓            |
+| Can request rework of RFCs<br> in represented area of responsibility                                                                        |            ✗            |            ✓           |            ✓            |
+| Can request rework of RFCs<br> in any part of the project                                                                                   |            ✗            |            ✗           |            ✓            |
+| Can manage release process of the project                                                                                                   |            ✗            |            ✗           |            ✓            |
+| Can represent the project in public as a Maintainer                                                                                         |            ✗            |            ✗           |            ✓            |
+
+These roles are merit based. Refer to the corresponding section for specific
+requirements and the nomination process.
+
+## Contributor
+
+A Contributor invests time and resources to improve oneTBB project.
+Anyone can become a Contributor by bringing value in one of the following ways:
+  * Answer questions from community members.
+  * Propose changes to the design.
+  * Provide feedback on design proposals.
+  * Review and/or test pull requests.
+  * Test releases and report bugs.
+  * Contribute code, including bug fixes, features implementations,
+and performance optimizations.
+
+## Code Owner
+
+A Code Owner has responsibility for a specific project component or a functional
+area. Code Owners are collectively responsible, with other Code Owners,
+for developing and maintaining their component or functional areas, including
+reviewing all changes to corresponding areas of responsibility and indicating
+whether those changes are ready to be merged. Code Owners have a track record of
+contribution and review in the project.
+
+Requirements:
+  * Track record of accepted code contributions to a specific project component.
+  * Track record of contributions to the code review process.
+  * Demonstrated in-depth knowledge of the architecture of a specific project
+    component.
+  * Commits to being responsible for that specific area.
+
+The process of becoming a Code Owner is:
```

Comment:
I would simplify it: **How to become a Code Owner?** or **To become a Code Owner:**